### ISSUE TITLE
CVE-2019-1551 says "x64_64" instead of "x86_64"

### ISSUE THREAD

**Issue by DerDakon** [openssl/openssl#12977]
This affects both https://www.openssl.org/news/vulnerabilities.html#y2019 as well as the entries in CHANGES.md and NEWS.md.

**Comment by nhorman** [comment]
marking as inactive, to be closed at the end of 3.4 dev, barring further input

**Comment by DerDakon** [comment]
what input would you need? Just change the string to match the actual platform name, no?

**Comment by nhorman** [comment]
yes, but in this case, I was looking for a statement from someone that still considers this, I suppose for lack of a better term, relevant.  The entry being referred to is over 6 years old now, so I'm biased toward thinking its not something the core development team is likely to get to.  However, if you do have interest in seeing it fixed, I'm happy to mark this as a community issue, which we would appreciate a PR for.  Are you interested in handling that work?  If so, I'll assign this to you to take care of.

**Comment by DerDakon** [comment]
I can do PRs for that if I can access the relevant repos.

**Comment by nhorman** [comment]
@DerDakon thank you!  As a community participant you have access to everything you need to move this forward.  We follow the standard github workflow for community members to create pull requests to make changes to our code:

1) Start by going to the home page for the openssl repository [here](https://github.com/openssl/openssl)
2) in the upper right area of the screen press the fork button, and follow the prompts to create a personal writeable copy of the openssl tree for your use.
3) From your newly forked copy, click the green code button to reveal the url to use for cloning your personal copy of the openssl tree to your local system
4) In your newly cloned local copy, use whatever tools/ides you need to do the following
    a) Create and checkout a branch to make your needed changes on.  Name the branch whatever you like
    b) Make the changes you are interested in.  In this case the offending data you want to change will be found in the NEWS.md file in the root directory of the tree
    c) commit your changes locally, mention this issue in the commit message for your change in the form:
```
Fixes #12977
``` 
As that will automatically associate this issue with your requested change.  Push your branch back to your personal copy of the openssl tree.

 5) go back to your personal fork of your tree on the github, website (from step 3 above), and under the pull request tab, click the new pull request tab.  Follow the prompts to create a pull request.  You're base should be openssl/master, and your source should be your newly pushed branch.  click the button to finish creating the pull request.

Note, normally changes require a contributors license agreement to allow code to be contributed, but in this even, since you're just requesting a typo, that can be skipped by applying the "cla:trivial" label to the pull request you created in 4(d) above, which you are welcome to do yourself

From there, core team members will review, approve and merge your change.
I'm assigning this as a community issue to you.  And thank you for being willing to help out here!

**Comment by mattcaswell** [comment]
@nhorman - this requires a fix in the security advisory and the associated json. Realistically I think this is something only we can do - so this probably needs to be assigned to a core dev team member. It's a simple fix.
